app/controllers/home_controller.rb

```diff
@@ -98,7 +98,7 @@ def activity
                    .page(params[:page])
                    .group(['title', 'comments.cid']) # ONLY_FULL_GROUP_BY, issue #3120
 
-    if logged_in_as(['admin', 'moderator'])
+    if logged_in_as((%w(moderator admin)))
```

Comment:
Hi @Dorcas-BD , my bad I included extra braces .. it should be `if logged_in_as (%w(moderator admin))`. please make the change, thanks

Reply:
but that shouldn't be affecting the tests though :thinking: